### ISSUE TITLE
Issue with multiple fields in SuperTable

### ISSUE THREAD

**Issue by br-designer** [engram-design/SuperTable#359]
### Description
I'm only able to access the first element within a SuperTable field using GraphQL. This is specific to a SuperTable nested within a matrix. 

### Steps to reproduce

1. Create a matrix with this structure: 
`Matrix Block > Matrix Type 1 > SuperTable > DropDown Field 1 && DropDown Field 2`
`Matrix Block > Matrix Type 2 > SuperTable > DropDown Field 1 && DropDown Field 2`

2. Use a schema similar to: 
`query($section: [String], $slug: [String]) {
    entries(section: $section, slug: $slug) {
        uri
        title
        postDate
        ... on pages_pages_Entry {
            contentBlocks {
                ... on contentBlocks_text_BlockType {
                    text
                    format {
                        ... on format_BlockType {
                            width 
                            height
                        }
                    }
                }
                ... on contentBlocks_image_BlockType {
                    image 
                    format {
                        ... on format_BlockType {
                            width
                            height
                        }
                    }
                }
            }
        }
    }
}`
3. "height" (or DropDown Field 2) on both blocks shows `"Cannot query field "height" on type "format_BlackType"`. 
### Additional info

- Plugin version: 2.4.9
- Craft version: 3.4.20

**Comment by engram-design** [comment]
Can you confirm you have a similar thing to the following structure and field setup?

See the block setup:
![Screen Shot 2020-05-31 at 4 32 18 pm](https://user-images.githubusercontent.com/1221575/83346010-5533d380-a35c-11ea-9cea-1ba6cf190f28.png)

The GQL query:
```
{
  entries (section:"superTableTesting") {
    ... on superTableTesting_superTableTesting_Entry {
      mstGql {
        ... on mstGql_block1_BlockType {
          superTable1 {
            ... on superTable1_BlockType {
              dropdownField1
              dropdownField2
            }
          }
        }
        
        ... on mstGql_block2_BlockType {
          superTable2 {
            ... on superTable2_BlockType {
              dropdownField12
              dropdownField22
            }
          }
        }
      }
    }
  }
}
```

And the result:
```
{
  "data": {
    "entries": [
      {
        "mstGql": [
          {
            "superTable1": [
              {
                "dropdownField1": "Some Value",
                "dropdownField2": "Some Value 2"
              }
            ]
          },
          {
            "superTable2": [
              {
                "dropdownField12": "Another Value",
                "dropdownField22": "Another Value 2"
              }
            ]
          }
        ]
      }
    ]
  }
}
```

Everything seems okay to me?

**Comment by br-designer** [comment]
I used your example and it outputs as expected. The original supertable I was having issues with now works as well. I had changed the field types right before this issue started. Maybe some caching or lag. Thanks for your help!

**Comment by jpcarpenter** [comment]
I'm actually encountering this same exact issue. Once I change the field type of a field within a Matrix->SuperTable layout, I can't access any other field located within the super table, with the exception of the first field. The only way I am able to get it to work again is by changing the super tables field handle to something else.  I've tried clearing every type of cache I could find, but it doesn't seem to be making any sort of impact. Suggestions greatly appreciated.

**Comment by engram-design** [comment]
Hmm, I wonder if this is down the to GQL cache Craft maintains. Will investigate

**Comment by jpcarpenter** [comment]
@engram-design any status on this? I'm running into this issue again on another Matrix->SuperTable layout. I've cleared all caches from the CP and Craft CLI with no luck.

**Comment by engram-design** [comment]
I can't replicate the issue sorry. I've tried adding, removing, re-arranging fields, changing their type and my example above seems to work pretty consistently.

**Comment by jpcarpenter** [comment]
@engram-design I think it has something to do with project config. When I hit the error again, I went and rebuilt the project config based on the data stored throughout the database, and I was finally able to access the missing fields within my GQL schema. Is that expected behavior, or am I just missing a core concept of project config and how it works with craft environments/plugins?

**Comment by engram-design** [comment]
Hmm, It _shouldn't_ make a difference, as the project config data should be updated whenever you save the field. It just seems very odd It's working for me. I'll do some more testing, but good to know a rebuild will fix this, that might help figuring this out.

**Comment by engram-design** [comment]
Fixed in [2.6.8](https://github.com/verbb/super-table/releases/tag/2.6.8)
